docs/basics/mutating-data.md

```diff
@@ -0,0 +1,53 @@
+---
+title: mutating-data
+order: 2
+---
+
+## (P)React
+
+Now that we know how to query our data we'll also need to know
+how to mutate it, for this the `useMutation` hook and the `Mutation`
+component are exported.
+
+Let's set up a mutation allowing us to change the name of our todo.
+
+```jsx
+const Todo = ({ id, title }) => {
+  const [updateTodoResult, updateTodo] = useMutation(`
+    mutation ($id: ID!, $title: String!) {
+      updateTodo (id: $id, title: $title) {
+        id
+      }
+    }
+  `);
+}
+```
+
+Similar to the `useQuery` output we see a tuple, the first being our result
```

Comment:
I've tried rewording this a tiny bit, what do you think? 🙂 

Similar to the `useQuery` output, `useMutation` returns a tuple. The first item in the tuple being our result
containing: `fetching`, `error`, and `data`. At this point in time, no mutation has been performed. To mutate the data we first have to invoke the second item in the tuple - the function here named `updateTodo`.